exo-run/features/run.feature

```diff
@@ -52,5 +52,5 @@ Feature: running Exosphere applications
 
 
   Scenario: a service crashes during startup
-    Given a running "crashing-service" application
+    When running a "crashing-service" application
```

Comment:
This app crashes, so we should say `When trying to run the "crashing-service" application`